src/k8sdeploy/k8sdeploy.py

```diff
@@ -22,9 +24,13 @@ def __init__(self,var_filename,stack,ecr_account_id):
     def __deploy_data__(self,filename,ecr_account_id):
         self.checkAWSToken(ecr_account_id)
         var_data=self.read_variable_file(filename)
+        # manual inputs
+        var_data['ecr_account_id'] = ecr_account_id
+        var_data['target_stack']=self.stack
+        #var_data['target_release_tag'] = self.release_tag[1:] if self.release_tag.lower().startswith('v') else self.release_tag
```

Comment:
in general I think it's best to remove commented-out code - we can always recover it from history if necessary